src/test/java/com/google/neighborgood/IntegrationTest.java

```diff
@@ -0,0 +1,1038 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.openqa.selenium.support.ui.ExpectedConditions.*;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.runners.MethodSorters;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Select;
+
+@RunWith(JUnit4.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
Ah, yes I can add some comments and will look into making the tests less co-dependent. I thought setting up things in the background for some of the tests was almost like cheating since it wasn't actually testing out how the user would interact with it :) but this sounds like a simpler approach, so I'll start making some of these changes next week.

Reply:
I'll make the changes to remove the FixMethodOrder decorator in another PR as this one seems pretty long run already. Would it be alright to eliminate the clearAllDatastoreEntities method altogether and have testers run `mvn clean` before the test instead? There's been 2-3 instances where the test suite has failed on my end on the first test because all task entities weren't successfully deleted with webdriver, so perhaps eliminating this external factor from the test would be better.